### PR TITLE
Modify OpenVINO and Streaming Python doc

### LINKED ISSUE #1353: (Python) Streaming example issues
Most issues in https://github.com/intel-analytics/analytics-zoo/issues/1342 are also in Python.

1. If we don't have a pre-trained model for text classification, I think we should just remove the example.

2. The readme of object detection example should be updated. 
```
In this example, we demo the model inference using a pre-trained SSD model and Spark Streaming. The example contains two programs:
1. ImagePathWriter simulates a streaming source, and periodically write image paths into text files in the image path folder.
2. StreamingObjectDetection continuously monitors the image path folder,  and periodically loads corresponding images, run object detection model on them,  and save the results in the output folder.
"
```

3. image_path_writer command is incorrect. Should use spark submit and the program parameters are wrong. And we should use longer delay and smaller batch by default.

MASTER=...
imageSourcePath=... // image path. Local file system/HDFS/Amazon S3 are supported
streamingPath=... // text files. Local file system/HDFS/Amazon S3 are supported
**${SPARK_HOME}/bin/spark-submit-with-zoo.sh** \
    --master ${MASTER} \
    --driver-memory 5g \
    image_path_writer.py \
    **--streamingPath ${streamingPath} --imageSourcePath ${imageSourcePath}**

### PR DESCRIPTION
**Pull request by qiyuangong**
Fix #1353.
Add System Requirements for #1374 .
Address #1368 .

### CODE REVIEW

**Comment by zhichao-li** [comment]
somehow link this doc and `https://github.com/intel-analytics/analytics-zoo/tree/master/pyzoo/zoo/examples/openvino` together otherwise the user might lack the context with the requirements for the example?

**Comment by zhichao-li** [comment]
related issue: https://github.com/intel-analytics/analytics-zoo/issues/1374

**Comment by hkvision** [comment]
Add to the README of python ov example?

File: pyzoo/zoo/examples/openvino/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ In this example, we use `frozen_inference_graph.pb` of the `faster_rcnn_resnet10
 ```bash
 export SPARK_DRIVER_MEMORY=10g
```
Comment (hkvision) [inline pyzoo/zoo/examples/openvino/README.md]: Also add system requirement and dependencies here?

**Pull request merged**